parsl/dataflow/dflow.py

```diff
@@ -370,10 +370,15 @@ def handle_app_update(self, task_id, future, memo_cbk=False):
         # If checkpointing is turned on, wiping app_fu is left to the checkpointing code
         # else we wipe it here.
         if self.checkpoint_mode is None:
-            self.tasks[task_id]['app_fu'] = None
-        self.tasks[task_id]['depends'] = []
+            self.wipe_task(task_id)
         return
 
+    def wipe_task(self, task_id):
+        """ Remove task with task_id from the internal tasks table
+        """
+        logger.warning("Wiping {}".format(task_id))
```

Comment:
this should be an info or a debug, not a warning.

Reply:
Sorry, I should've removed that.